Cell bender analysis

Srivalli-Kolla opened the pull request.

cartal commented:
This is very good. 

I have a couple of comments on the work: 

1) The names have the right naming, but it would be good to know what the difference between ambient and full is. Also, we need to know what is the final result. Which fdr value did you choose? 

2) Since we want to evaluate how well the ambient contamination was removed from the other non-myocyte cells, we need markers for those cell compartments to visualise them in the UMAP. I left some on the notebook.

Srivalli-Kolla commented:
Hi Carlos, 

I have updated the filenames, filtering , genes and all suggested changes in the commit. Could you please have a look and share your thoughts/suggestions . 

commit 9236d41c66e613a94e55d8b19db319f95a531da9
commit name : Corrected filenaming, filtration

Srivalli-Kolla commented:
Hi Carlos

I have added the updated figures with appropriate file naming and figures in following commit which I made recently

Commit id : 7a83a9ebbdedf5ef59a6f49007061e94391d5886
Commit name : Added new genes from empty droplets and additional plotting

Link for the commit : [Added new genes from empty droplets and additional plotting](https://github.com/Talavera-Lopez-Lab/hofmann_dmd/pull/12/commits/7a83a9ebbdedf5ef59a6f49007061e94391d5886)
